kmath-units/src/commonMain/kotlin/space/kscience/kmath/units/units.kt

```diff
@@ -0,0 +1,276 @@
+public val t: Measure = Measure(BaseUnits.KILOGRAM to 1, multiplier = 1000.0)
+public val Da: Measure = Measure(BaseUnits.KILOGRAM to 1, multiplier = 1.660_539_040_202_020 * 10e-27)
+public val eV: Measure = J.copy(multiplier = 1.602_176_634 * 10e-19)
+
+public fun Y(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e24)
+public fun Z(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e21)
+public fun E(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e18)
+public fun P(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e15)
+public fun T(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e12)
+public fun G(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e9)
+public fun M(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e6)
+public fun k(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e3)
+public fun h(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e2)
+public fun da(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e1)
+
+public fun y(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-24)
+public fun z(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-21)
+public fun a(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-18)
+public fun f(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-15)
+public fun p(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-12)
+public fun n(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-9)
+public fun u(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-6)
+public fun m(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-3)
+public fun c(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-2)
+public fun d(measure: Measure): Measure = measure.copy(multiplier = measure.multiplier * 1e-1)
+
+public infix fun Measure.pow(power: Int): Measure =
+    copy(chain = chain.mapValues { (_, v) -> IntRing.power(v, power.toUInt()) }, multiplier = multiplier.pow(power))
+
+public open class MeasurementAlgebra<T>(public open val algebra: Algebra<T>) : Algebra<Measurement<T>> {
+    public operator fun T.times(m: Measure): Measurement<T> = Measurement(m, this)
+    public operator fun Measurement<T>.times(m: Measure): Measurement<T> =
+        copy(measure = MeasureAlgebra { measure * m })
+
+    public override fun bindSymbol(value: String): Measurement<T> = algebra.bindSymbol(value) * pure
+}
+
+public fun <T> Algebra<T>.measurement(): MeasurementAlgebra<T> = MeasurementAlgebra(this)
+
+public open class MeasurementSpace<T, out A>(public override val algebra: A) : MeasurementAlgebra<T>(algebra),
```

Comment:
What do you think about adding function to converting from one measurement to another, e.g.:
```
fun Measurement<T>.convertTo(measure: Measure): Measurement<T> {
    require(this.measure.chain == measure.chain) {...}
    return Measurement (
        measure = measure,
        value = this.value * this.measure.multiplier / measure.multiplier
    )
}
```